### ISSUE TITLE
rivinec wallet send coins broken: does not accept default unit

### ISSUE THREAD

**Issue by GlenDC** [rivine/rivine#177]
According to `rivinec wallet send coins`:

> Send coins to an address. 'dest' must be a 76-byte hexadecimal address.
> 'amount' can be specified in units, e.g. 1.23K. Run 'wallet --help' for a list of units.
> If no unit is supplied, hastings will be assumed.

However, when running for example `rivinec wallet send coins 1 67b7cd6353774b0a90618d17efa3c869d6ef40773ed17298b8a34aa8f4d964659672c382481e` I get the following error:

```
Could not parse amount: amount is missing units; run 'wallet --help' for a list of units
```

Meaning it does not parse a default unit. As I think having a default unit is a sane thing to do, and using the smallest unit when doing so is very reasonable as well. Hence the documentation is to be considered correct here and the implementation should be (bug) fixed, such that it uses a default unit when none can be found.

**Comment by robvanmieghem** [comment]
Let's just drop the hastings documentation, no-one wants to use that term anyway

**Comment by GlenDC** [comment]
Are you just focussing on the name `hasting` or you want to drop support for units other than the main coin?

**Comment by robvanmieghem** [comment]
yes, hate the hastings name  and for now, the easiest thing to do would be to modify the docs to say the unit is required since we might want to rethink  and rename the units anyway.

**Comment by GlenDC** [comment]
I don't mind, that's a client wallet change anyhow, so perhaps for now simply modify the documentation and ensure that the function doesn't do anything else wrong, besides that unit thingie.

**Comment by GlenDC** [comment]
Closed via 499e6f1.
